src/import/common.rs

```diff
@@ -0,0 +1,205 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::ops::Deref;
+use std::sync::Arc;
+
+use kvproto::metapb::*;
+use kvproto::kvrpcpb::*;
+use kvproto::importpb::*;
+
+use pd::RegionInfo;
+
+use super::client::*;
+
+// Just used as a mark, don't use them in comparison.
+pub const RANGE_MIN: &[u8] = &[];
+pub const RANGE_MAX: &[u8] = &[];
+
+pub fn new_range(start: &[u8], end: &[u8]) -> Range {
+    let mut range = Range::new();
+    range.set_start(start.to_owned());
+    range.set_end(end.to_owned());
+    range
+}
+
+pub fn before_end(key: &[u8], end: &[u8]) -> bool {
+    key < end || end == RANGE_MAX
+}
+
+pub fn inside_region(key: &[u8], region: &Region) -> bool {
```

Comment:
We already have function `check_key_in_region`, can we reuse it here?

Reply:
But I want it to return a bool, and it is a simple function, so I don't bother to reuse it.